### PR TITLE
removes unused secrets panel button, fixes a button in the global player panel

### PR DESCRIPTION
**Pull request by nervere**
## About The Pull Request

check_antagonist topic for secrets panel had some unused code that did nothing, and it also didn't check antagonists.

## Changelog
:cl: Nervere
admin: removed a duplicate/broken secrets panel button
admin: you can now click the text that says "Antagonist" on an antagonist's entry in the global player panel to check all antags, as intended
/:cl:

### CODE REVIEW

**Comment by AnturK** [comment]
I'd honestly remove this, no need to have two access points.

**Comment by nervere** [comment]
Sure, I'll do that instead.

**Comment by nervere** [comment]
I removed something that I shouldn't have in topic.dm, I'm going to add part of it back so I don't break a few buttons. The button in the secrets panel will be removed, though, as it's duplicate and largely useless. Also, the button in the player panel should now work regardless.

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -228,7 +228,6 @@
 						message_admins("<span class='adminnotice'>[key_name_admin(usr)] called the Emergency Shuttle to the station.</span>")
 
 
-		href_list["secrets"] = "check_antagonist"
```
Comment (AnturK) [inline code/modules/admin/topic.dm]: What was this nonsense.

**Pull request merged**